spec/adapters/will_paginate_already_paginated_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require 'spec_helper'
+RSpec.describe Wor::Paginate::Adapters::KaminariAlreadyPaginated do
+  describe '#index' do
+    let!(:n) { 28 }
+    let!(:n_page) { 10 }
+    let!(:dummy_models) { create_list(:dummy_model, n) }
+    context 'when paginating something already paginated' do
+      context 'with no results' do
```

Comment:
Everything here returns paginated data... So, What does the `with no results` mean?